converters/google/convert.go

```diff
@@ -86,13 +87,16 @@ type AssetResource struct {
 }
 
 // NewConverter is a factory function for Converter.
-func NewConverter(resourceManager *cloudresourcemanager.Service, project, ancestry, credentials string) (*Converter, error) {
+func NewConverter(resourceManager *cloudresourcemanager.Service, project, ancestry, credentials string, offline bool) (*Converter, error) {
 	cfg := &converter.Config{
 		Project:     project,
 		Credentials: credentials,
 	}
-	if err := cfg.LoadAndValidate(); err != nil {
-		return nil, errors.Wrap(err, "configuring")
+
+	if !offline {
```

Comment:
have you tried removing `LoadAndValidate()` invocation altogether? Since we don't expect any API calls to be made, it doesn't seem to make sense to initialize those clients.

Reply:
I created another pull request at #39 for removing `LoadAndValidate()`.